uploader/src/test/kotlin/com/cloudinary/UploaderTest.kt

```diff
@@ -869,6 +869,48 @@ class UploaderTest(networkLayer: NetworkLayer) {
         assertTrue(doResourcesHaveTag(cloudinary, tag4, publicId1))
     }
 
+    @Test
+    fun testTagsAsArray() {
```

Comment:
It would be easier to just write a simple unit test, instead of complex integration tests that are less reliable.